src/inc/DefaultSettings.h

```diff
@@ -42,4 +42,5 @@ constexpr COLORREF DEFAULT_CURSOR_COLOR = COLOR_WHITE;
 constexpr COLORREF DEFAULT_CURSOR_HEIGHT = 25;
 
 const std::wstring DEFAULT_WORD_DELIMITERS{ L" ./\\()\"'-:,.;<>~!@#$%^&*|+=[]{}~?\u2502" };
+constexpr int32_t DEFAULT_SNAP_ON_OUTPUT = 0x1 | 0x2;

```

Comment:
I don't like this but I don't know how to fix it. Or is it ok if I just remove it?

Reply:
just remove it i thought